<ui version="4.0" >
 <class>NavigationBarWidgetClass</class>
 <widget class="QWidget" name="NavigationBarWidgetClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>76</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>NavigationBarWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <property name="leftMargin" >
    <number>2</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>2</number>
   </property>
   <property name="bottomMargin" >
    <number>2</number>
   </property>
   <property name="spacing" >
    <number>1</number>
   </property>
   <item row="1" column="0" colspan="6" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>274</width>
       <height>29</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="6" >
    <widget class="QPushButton" name="SlidePushButton" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>40</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>40</width>
       <height>32</height>
      </size>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset>
       <normalon>:/images/slide-vertical.png</normalon>
      </iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>40</width>
       <height>32</height>
      </size>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QPushButton" name="BackPushButton" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="autoFillBackground" >
      <bool>false</bool>
     </property>
     <property name="styleSheet" >
      <string notr="true" />
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset>
       <normalon>:/images/back.png</normalon>
      </iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QPushButton" name="HomePushButton" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="autoFillBackground" >
      <bool>false</bool>
     </property>
     <property name="styleSheet" >
      <string notr="true" />
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset>
       <normalon>:/images/home.png</normalon>
      </iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QPushButton" name="ForwardPushButton" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="autoFillBackground" >
      <bool>false</bool>
     </property>
     <property name="styleSheet" >
      <string notr="true" />
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset>
       <normalon>:/images/forward.png</normalon>
      </iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="3" >
    <widget class="Line" name="line" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>15</width>
       <height>40</height>
      </size>
     </property>
     <property name="autoFillBackground" >
      <bool>false</bool>
     </property>
     <property name="styleSheet" >
      <string notr="true" />
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="4" >
    <widget class="QPushButton" name="ZoomOutPushButton" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="autoFillBackground" >
      <bool>false</bool>
     </property>
     <property name="styleSheet" >
      <string notr="true" />
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset>
       <normalon>:/images/zoom-out.png</normalon>
      </iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="5" >
    <widget class="QSlider" name="ZoomSlider" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="autoFillBackground" >
      <bool>false</bool>
     </property>
     <property name="styleSheet" >
      <string notr="true" />
     </property>
     <property name="minimum" >
      <number>50</number>
     </property>
     <property name="maximum" >
      <number>150</number>
     </property>
     <property name="singleStep" >
      <number>25</number>
     </property>
     <property name="pageStep" >
      <number>25</number>
     </property>
     <property name="value" >
      <number>100</number>
     </property>
     <property name="tracking" >
      <bool>false</bool>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance" >
      <bool>false</bool>
     </property>
     <property name="invertedControls" >
      <bool>false</bool>
     </property>
     <property name="tickPosition" >
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval" >
      <number>25</number>
     </property>
    </widget>
   </item>
   <item row="2" column="6" >
    <widget class="QPushButton" name="ZoomInPushButton" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="autoFillBackground" >
      <bool>false</bool>
     </property>
     <property name="styleSheet" >
      <string notr="true" />
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset>
       <normalon>:/images/zoom-in.png</normalon>
      </iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="6" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="images.qrc" />
 </resources>
 <connections/>
</ui>
